vendor/github.com/ethereum/go-ethereum/p2p/discv5/metrics.go

```diff
@@ -0,0 +1,8 @@
+package discv5
+
+import "github.com/ethereum/go-ethereum/metrics"
+
+var (
+	ingressTrafficMeter = metrics.NewMeter("discv5/InboundTraffic")
```

Comment:
Why `egress` = `Inbound` and `egress` = `Outbound`? Would prefer consistent naming internal and external.

Reply:
this is consistent with p2p metrics https://github.com/ethereum/go-ethereum/blob/master/p2p/metrics.go#L28-L31

Reply:
IC, thanks. Still feels strange. 😁